octoprint_mrbeam/support.py

```diff
@@ -21,13 +23,14 @@ def check_support_mode(plugin):
     :returns True if support_mode is enabled, False otherwise
     """
     support_mode_enabled = False
+    if not hasattr(plugin, "_settings") or not hasattr(plugin, "_user_manager"):
+        return None
+
     try:
         if plugin._settings.get(["dev", "support_mode"]) or (
             os.path.isfile(SUPPORT_STICK_FILE_PATH)
-            and time.time() - os.path.getmtime(SUPPORT_STICK_FILE_PATH)
-            < SUPPORT_STICK_FILE_MAX_AGE
+            and len(os.listdir(USB_MOUNT_DIR)) > 0  # if i a usb stick is plugged in
```

Comment:
Use [`os.path.isdir()`](https://docs.python.org/3/library/os.path.html#os.path.isdir)

Reply:
Nope. I want the dirs in there. The number of dirs must be 1 or more!

Reply:
No. I want to test if the number of subdirs in there is at least 1.